synapse/storage/databases/main/account_data.py

```diff
@@ -94,7 +94,10 @@ def __init__(
                 hs.get_replication_notifier(),
                 "room_account_data",
                 "stream_id",
-                extra_tables=[("room_tags_revisions", "stream_id")],
+                extra_tables=[
+                    ("account_data", "stream_id"),
+                    ("room_tags_revisions", "stream_id"),
+                ],
```

Comment:
Should this include room_account_data too, like the postgres ID generator?

Reply:
That's included above. (These are `extra_tables=` rather than `tables=` that `MultiWriterIdGenerator` uses)

Reply:
(This feels like something a lint could check 😢 )